src/styles/blocks/article.css

```
@@ -0,0 +1,146 @@
+.article {
+    font-size: 12px;
+    line-height: 130%;
+    letter-spacing: -0.02em;
+}
+
+.article__heading-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin: 0 -8px 16px;
+    color: white;
+    background-color: var(--color-grey-dark);
+}
+
+.article__heading-wrapper::after {
+    content: ' ';
+    display: block;
+    align-self: center;
+    width: 320px;
+    height: 170px;
+    background-image: url('../../images/patterns/dots.svg');
```

Сделать абсолютный путь 

```suggestion
    background-image: url('/images/patterns/dots.svg');
```